### PR TITLE
fix: traffic-metrics example

### PR DESCRIPTION
**Pull request by lahabana**
<Explain your change!>

Did you sign your commit? [Instructions](https://github.com/kumahq/.github/blob/main/CONTRIBUTING.md#sign-your-commits)

Have you read [Contributing guidelines](https://github.com/kumahq/.github/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: app/_src/policies/traffic-metrics.md [hunk 1]
```diff
@@ -54,7 +54,8 @@ spec:
     - name: prometheus-1
       type: prometheus
       conf:
-        skipMTLS: false
```
Comment (lukidzi) [inline app/_src/policies/traffic-metrics.md:57]: We should fix the example in line 102 for universal which configures tls disabled but here we create tls enabled.

File: app/_src/policies/traffic-metrics.md [hunk 2]
```diff
@@ -54,7 +54,8 @@ spec:
...
+        tls:
+          mode: activeMTLSBackend
```
Comment (slonka) [inline app/_src/policies/traffic-metrics.md:58]: with this change the <= 2.3 version is the same as >= 2.4 version... it does not make sense for me

**Comment by lahabana** [comment]
@slonka  and @lukidzi fixed. This doc is very hard to follow :)

**Review by lukidzi — APPROVED** [review]
